scripts/review.js

```diff
@@ -35,7 +35,7 @@ module.exports = (robot) => {
       'url'
     ])
     .then((response) => {
-      _.forEach(response.body, game => res.send(`*${game.name}* with the rating of ${Math.round(game.rating) / 10}\n${game.summary}\n\nCover: ${game.cover.url}\nURL: ${game.url}`));
+      _.forEach(response.body, game => res.send(`*${game.name}* with the rating of ${Math.round(game.rating) / 10}\n${game.summary}\n\nCover: ${game.cover.url}\nURL: ${game.url}`)); // eslint-disable-line max-len
```

Comment:
This one is totally my fault. Not sure why the linter didn't creep up sooner, ie. my machine nor travis. 

I'm sure we can split that into few lines as oppose to hack linter :D

```js
_.forEach(
  response.body,
  game => {
    const msg = `*${game.name}* with the rating of ${Math.round(game.rating) / 10}
${game.summary}

Cover: ${game.cover.url}
URL: ${game.url}`;

    return res.send(msg));
```

Reply:
If we do it rarely, I'm okay with letting it slide. I didn't even know this line was offending the linter until I ran it. I'm not opposed to letting it stay, vim does nice line wrapping for me. 

If I was going to change it, I'd rather do something like this

```javascript
_.forEach(response.body, game => res.send([
  `*${game.name}* with the rating of ${Math.round(game.rating) / 10}`,
  `${game.summary}`,
  '',
  `Cover: ${game.cover.url}`,
  `URL: ${game.url}`
].join('\n')));
```